modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java

```diff
@@ -470,6 +492,19 @@ private int _getLowestPriority(
 		);
 	}
 
+	private int _getHighestPriority(
+		long groupId, long classNameId, long classPK) {
+
+		return Optional.ofNullable(
+			segmentsExperiencePersistence.fetchByG_C_C_A_First(
+				groupId, classNameId, classPK, true, null)
```

Comment:
Again, in order to get the highest priority you should not filter by active.

Try this:
1. Create AB Test
2. Create 2 variants

Reply:
Hey!!!, that is totally true, I thought I have generated a new method, sorry :)

Reply:
Ohh!!!, I did it in today's Brian's pull. I will change it on this pull after I merge those changes.

Awesome!!!